### PR TITLE
Fixed misleading info about cross-shard miniblocks processing

### PR DESCRIPTION
**Pull request by SebastianMarian**
* Fixed misleading info about number of txs processed for a cross shard miniblock when an error is occured ("time is out" or any other error)

### CODE REVIEW

**Review by sasurobert — DISMISSED** [review]

File: process/block/preprocess/rewardTxPreProcessor.go [hunk 1]
```diff
@@ -422,35 +422,35 @@ func (rtp *rewardTxPreprocessor) CreateAndProcessMiniBlocks(
...
 	if rtp.blockSizeComputation.IsMaxBlockSizeWithoutThrottleReached(1, len(miniBlockRewardTxs)) {
-		return nil, process.ErrMaxBlockSizeReached
+		return nil, 0, process.ErrMaxBlockSizeReached
 	}
 
 	processedTxHashes := make([][]byte, 0)
 	for index := range miniBlockRewardTxs {
 		if !haveTime() {
-			return processedTxHashes, process.ErrTimeIsOut
+			return processedTxHashes, len(processedTxHashes), process.ErrTimeIsOut
```
Comment (ccorcoveanu) [inline process/block/preprocess/rewardTxPreProcessor.go:446]: Maybe I'm missing something, but isn't the second returned parameter always `len(processedTxHashes)` (or `index` in the following cases below)?

Comment (SebastianMarian) [reply process/block/preprocess/rewardTxPreProcessor.go]: Yes, you are right

**Review by sasurobert — APPROVED** [review]

**Review by sasurobert — APPROVED** [review]

**Review by ccorcoveanu — APPROVED** [review]

**Review by mateigraura — APPROVED** [review]
System test passed 👍🏻

**Review by valentincraciun99 — APPROVED** [review]
System test passed

**Pull request merged**